www/data_service/src/classes/builder.service.coffee

```diff
@@ -0,0 +1,17 @@
+class Builder extends Factory
+    constructor: (Base, dataService) ->
```

Comment:
how about:

     return class BuilderInstance extends Base
                endpoints: [
                    'builds'            # /builds/:buildid
                    'buildrequests'     # /buildrequests/:buildrequestid
                    'forceschedulers'   # /forceschedulers
                    'buildslaves'       # /buildslaves/:buildslaveid
                                        # /buildslaves/:name
                    'masters'           # /masters/:masterid
                ]


and he default constructor takes care of building the endpoints implicitly